drivers/flash/nrf_qspi_nor.c

```diff
@@ -381,22 +385,41 @@ static int qspi_send_cmd(const struct device *dev, const struct qspi_cmd *cmd,
 }
 
 /* RDSR wrapper.  Negative value is error. */
-static int qspi_rdsr(const struct device *dev)
+static int qspi_rdsrX(const struct device *dev, uint8_t sr_num)
```

Comment:
this doesn't follow guidelines: `only_lowercase_please`

Reply:
Fixed. Also fixed qspi_wrsrX -> qspi_wrsr